cmd/goal/clerk.go

```diff
@@ -338,6 +353,10 @@ var sendCmd = &cobra.Command{
 		if err != nil {
 			reportErrorf(errorConstructingTX, err)
 		}
+		if (rekeyTo != basics.Address{}) {
```

Comment:
nit : There is a IsZero function that would do just that.

Reply:
Fixed!